zhaquirks/thirdreality/curtain_gen2.py

```diff
@@ -0,0 +1,69 @@
+"""Third Reality Blind Gen2 devices."""
+
+from typing import Final
+
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import QuirkBuilder
+import zigpy.types as t
+from zigpy.zcl.foundation import BaseAttributeDefs, ZCLAttributeDef
+
+
+class THIRD_REALITY_Blind_Gen2_CLUSTER(CustomCluster):
+    """Third Reality's Blind Gen2 private cluster."""
+
+    cluster_id = 0xFFF1
+
+    class AttributeDefs(BaseAttributeDefs):
+        """Define the attributes of a private cluster."""
+
+        enable_disable_pir_remote: Final = ZCLAttributeDef(
+            id=0x0000,
+            type=t.uint8_t,
+            is_manufacturer_specific=True,
+        )
+
+        compensation_speed: Final = ZCLAttributeDef(
+            id=0x0001,
+            type=t.int8s,
+            is_manufacturer_specific=True,
+        )
+
+        limit_position: Final = ZCLAttributeDef(
+            id=0x0002,
+            type=t.uint16_t,
+            is_manufacturer_specific=True,
+        )
+
+
+(
+    QuirkBuilder("Third Reality, Inc", "3RSB02015Z")
+    .replaces(THIRD_REALITY_Blind_Gen2_CLUSTER, endpoint_id=1)
+    .switch(
+        attribute_name=THIRD_REALITY_Blind_Gen2_CLUSTER.AttributeDefs.enable_disable_pir_remote.name,
+        cluster_id=THIRD_REALITY_Blind_Gen2_CLUSTER.cluster_id,
+        force_inverted=True,
+        translation_key="enable_disable_pir_mode",
+        fallback_name="Enable/Disable PIR Remote",
+    )
+    .number(
+        attribute_name=THIRD_REALITY_Blind_Gen2_CLUSTER.AttributeDefs.compensation_speed.name,
+        cluster_id=THIRD_REALITY_Blind_Gen2_CLUSTER.cluster_id,
+        endpoint_id=1,
+        min_value=-100,
+        max_value=100,
+        step=1,
```

Comment:
For `number` entities, try adding `mode="box"` and see if you prefer how that makes the entity show up in HA:
```suggestion
        step=1,
        mode="box",
```
You won't get a slider then, but a "box" to input the number into instead. If you prefer this for these entities, feel free to also add it to the other number entity in this PR (or other PRs).